### PR TITLE
Cleanup code warnings

### PR DESCRIPTION
**Pull request by wcekan**
## Description
Cleanup active alerts
https://lgtm.com/projects/g/yahoo/elide/alerts/

## Motivation and Context
Code quality

## How Has This Been Tested?
Unit testing

## License
I confirm that this contribution is made under an Apache 2.0 license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by moizarafat — DISMISSED** [review]

File: elide-model-config/src/main/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidator.java [hunk 1]
```diff
@@ -194,6 +194,9 @@ public void readConfigs() throws IOException {
         readConfigs(fileLoader.loadResources());
     }
 
+    /**
+     * @throws IOException
```
Comment (wcekan) [inline elide-model-config/src/main/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidator.java:198]: @aklish is there a reason this function throws IOException when currently it does not?

File: elide-model-config/src/main/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidator.java [hunk 2]
```diff
@@ -204,6 +207,9 @@ public void readConfigs(Map<String, ConfigFile> resourceMap) throws IOException
         populateInheritance(this.elideTableConfig);
     }
 
+    /**
+     * @throws IOException
```
Comment (wcekan) [inline elide-model-config/src/main/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidator.java:211]: @aklish ditto

File: elide-model-config/src/main/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidator.java (hunk 1 above)
Comment (aklish) [reply elide-model-config/src/main/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidator.java:198]: No

File: elide-model-config/src/main/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidator.java (hunk 2 above)
Comment (aklish) [reply elide-model-config/src/main/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidator.java:211]: No

**Review by moizarafat — DISMISSED** [review]

**Review by moizarafat — APPROVED** [review]

**Pull request merged**